docs/python_modules/enchantfilters/SWFFilters.py

```diff
@@ -46,8 +47,18 @@ class VersionStringFilter(Filter):
     versionchars = string.digits + "."
 
     def _skip(self, word):
-        (version,sep,revparse) = word.partition('-')
+        (version,sep,rest) = word.partition('-')
         if version == word:
             return False # probably not a version string then
         else:
```

Comment:
Try removing this for easier reading

Reply:
I kept this one actually but I removed the others.